src/id.rs

```diff
@@ -83,53 +83,54 @@ impl Id {
 
 #[cfg(test)]
  mod test{
-//     use super::*;
-//     use sodiumoxide::crypto;
-//     use NameType;
-//     use test_utils::Random;
-//
-//     #[test]
-//     fn construct_id_with_keys() {
-//       let sign_keys = crypto::sign::gen_keypair();
-//       let asym_keys = crypto::box_::gen_keypair();
-//
-//       // let public_keys = (sign_keys.clone().0, asym_keys.clone().0);
-//       // let secret_keys = (sign_keys.clone().1, asym_keys.clone().1);
-//
-//       let id = Id::with_keys(sign_keys.clone(), asym_keys.clone());
-//
-//       assert_eq!(NameType::new(crypto::hash::sha512::hash(&sign_keys.0[..]).0),
-//           id.name());
-//       assert_eq!(&sign_keys.0, &id.signing_public_key());
-//       // FIXME(ben) 20/07/2015 once PartialEq is implemented for the private key, avoid slice
-//       assert_eq!(&sign_keys.1[..], &id.signing_private_key()[..]);
-//       assert_eq!(&asym_keys.0, &id.encrypting_public_key());
-//     }
-//
-//     #[test]
-//     fn assign_relocated_name_id() {
-//         let before = Id::new();
-//         let original_name = before.name();
-//         assert!(!before.is_relocated());
-//         let relocated_name: NameType = Random::generate_random();
-//         let mut relocated = before.clone();
-//         relocated.assign_relocated_name(original_name.clone());
-//
-//         assert!(relocated.assign_relocated_name(relocated_name.clone()));
-//
-//         assert!(!relocated.assign_relocated_name(relocated_name.clone()));
-//         assert!(!relocated.assign_relocated_name(Random::generate_random()));
-//         assert!(!relocated.assign_relocated_name(original_name.clone()));
-//
-//
-//         assert!(relocated.is_relocated());
-//         assert_eq!(relocated.name(), relocated_name);
-//         assert!(before.name()!= relocated.name());
-//         assert_eq!(before.signing_public_key(), relocated.signing_public_key());
```

Comment:
old 113: is tested in new 110 (dedicated test for fn is_relocated())
old 118: is tested in new 130
old 120: is tested in new 133
old 121: is tested in new 134
old 125: is tested in new 114
old 126: was redundant to old 101 (or should be at least in own unit test)
old 127: correct, should be tested as well (change actually happened)
old 128-132: correct, in order to assure that method does not change value of other properties

Thanks for your feedback!